### PR TITLE
Update README

### LINKED ISSUE #82: Un-deprecate this repo
As significant time has gone by since we migrated content from this repo to z2jh, it may be helpful to undeprecate the repo, update the README, and set the default branch to gh-pages.

cc/ @yuvipanda @choldgraf @minrk

### PR DESCRIPTION
**Pull request by willingc**
Closes #82.

I've made this PR against master since a README on gh-pages would require a long scroll to read. This should be okay since ultimately we will create a table in #83.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,41 @@
-# Deprecated
+# [helm-chart](https://github.com/jupyterhub/helm-chart)
 
-This repository has been merged into the [Zero to JupyterHub](https://github.com/jupyterhub/zero-to-jupyterhub-k8s) repository.
+[![Join the chat at https://gitter.im/jupyterhub/binder](https://badges.gitter.im/jupyterhub/binder.svg)](https://gitter.im/jupyterhub/binder?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
+
+This repository contains the Helm charts for BinderHub and JupyterHub. The
```
Comment (yuvipanda) [inline README.md]: Perhaps say 'Helm chart tarball repository'? Just to make it clear that actual development doesn't happen here, just the built tarballs

Comment (willingc) [reply README.md]: Good point!

**Comment by yuvipanda** [comment]
w00t

**Comment by willingc** [comment]
Thanks!

**Pull request merged**